docs/user/neuralnet.rst

```diff
@@ -498,5 +529,5 @@ total loss:
             loss += self.lambda1 * sum([w.abs().sum() for w in self.module_.parameters()])
             return loss
 
-*Note*: This example also reguralizes the biases, which you typically
- don't need to do.
+.. note:: This example also reguralizes the biases, which you typically
```

Comment:
While you're at it, could you please fix the "reguralizes" typo? (That probably was me)